luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
+    if date_format != 'epoch':
+        return datetime.datetime.strptime(date_string, date_format).timestamp()
```

Comment:
(Or wrap it at line 153, where if it catches a ValueError getting raised out of this function, it deletes the "date" field.  Probably the same thing in the end.)